test/util/preload.js

```diff
@@ -1,6 +1,6 @@
-/* global jest:false, expect:false */
+/* global jest:false, expect:false *
```

Comment:
@nickpalmer block comment needs to be closed here :-)

Reply:
Oops! Fixed in 4d882d2